components/OverflowMenu/react.js

```diff
@@ -35,21 +36,27 @@ export class OverflowMenu extends Component {
 
   render() {
     const { displayOverflowMenuActions } = this.state
-    const { children } = this.props
+    const { children, variant } = this.props
     return (
-      <div className={cx('overflow-menu')}>
+      <div
+        className={cx('overflow-menu', {
+          'overflow-menu--dark': ['secondary', 'dark'].includes(variant),
```

Comment:
@tarikhamilton should we account for this new prop in this component's tests? I believe we test for conditional classes elsewhere in MDL.

Reply:
I don't think it is totally necessary.

Reply:
While I would agree, just wanted to call it out for project consistency. We may want to discuss testing expectations in our next MDL meeting.